ledger/participant-integration-api/src/main/scala/db/migration/translation/ValueSerializer.scala

```diff
@@ -39,6 +39,7 @@ private[migration] object ValueSerializer {
         logger.error(
           s"*** Please upgrade your sandbox database by upgrading your SDK to 1.6 first. ***"
         )
+      case _ =>
```

Comment:
@remyhaemmerle-da: When running this migration on an index DB from last August, this match failed on the `Right` side. I only could get it to pass after add this wildcard match. Would this match your expectation of how this should work?

Reply:
That is the only think that make sens.  This should need to be backported, shouldn't  it ?

Reply:
I propose to simplify this unnecessary complexity of mine:

```scala

  private[this] val DeprecatedValueVersions = Set("1", "2", "3", "4", "5")
  private[this] val UnsupportedErrorMessage = """Unsupported transaction version '(\d)'""".r

  private[translation] def handleDeprecatedValueVersions[X](x: Either[ValueCoder.DecodeError, X]): Either[ValueCoder.DecodeError, X] = {
    x match {
      case Left(ValueCoder.DecodeError(UnsupportedErrorMessage(version))) if DeprecatedValueVersions(version) =>
        logger.error(
          s"*** Deserialization of value version $version is not supported by the SDK 1.7.0 or later. ***"
        )
        logger.error(
          s"*** Please upgrade your sandbox database by upgrading your SDK to 1.6 first. ***"
        )
      case _ => 
    }
    x
  }
```